app/Actions/Backup/ManageBackupFile.php

```
@@ -29,6 +32,12 @@ public function delete(BackupFile $file): void
         $file->status = BackupFileStatus::DELETING;
         $file->save();
 
+        SocketEvent::dispatch(new SocketEventDTO(
+            projectId: $file->backup->server->project_id,
+            type: 'backup-file.updated',
+            data: new BackupFileResource($file),
+        ));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Events\SocketEvent' in method 'delete'. (undefined)

(StaticAccess)